src/WebServerException/WebServerException.cpp

```diff
@@ -0,0 +1,50 @@
+#include "WebServerException.hpp"
```

Comment:
I think that you forget to add the header to the commit

Reply:
True